### ISSUE TITLE
Unhandled Exception: PlatformException(error, java.lang.IllegalStateException: Trying to create a platform view of unregistered type: net.touchcap ture.qr.flutterqr/qrview [BUG] 

### ISSUE THREAD

**Issue by HeySreelal** [juliuscanute/qr_code_scanner#252]
**Bug Description**

I have a ScanScreen() which contains the QrView. The app works perfectly fine on debug mode. But when I run the release mode: 
`flutter run --release`
the ScanScreen() is just a blank page and throws this exception: 

```
E/flutter (12430): [ERROR:flutter/lib/ui/ui_dart_state.cc(177)] Unhandled Exception: PlatformException(error, java.lang.IllegalStateException: Trying to create a platform view of unregistered type: net.touchcap
ture.qr.flutterqr/qrview
E/flutter (12430):      at io.flutter.plugin.platform.k$a.d(Unknown Source:229)
E/flutter (12430):      at io.flutter.embedding.engine.i.i$a.b(Unknown Source:152)
E/flutter (12430):      at io.flutter.embedding.engine.i.i$a.h(Unknown Source:144)
E/flutter (12430):      at d.a.c.a.j$a.a(Unknown Source:17)
E/flutter (12430):      at io.flutter.embedding.engine.e.b.e(Unknown Source:57)
E/flutter (12430):      at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(Unknown Source:4)
E/flutter (12430):      at android.os.MessageQueue.nativePollOnce(Native Method)
E/flutter (12430):      at android.os.MessageQueue.next(MessageQueue.java:336)
E/flutter (12430):      at android.os.Looper.loop(Looper.java:174)
E/flutter (12430):      at android.app.ActivityThread.main(ActivityThread.java:7711)
E/flutter (12430):      at java.lang.reflect.Method.invoke(Native Method)
E/flutter (12430):      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
E/flutter (12430):      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
E/flutter (12430): , null, null)
E/flutter (12430): #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:582)
E/flutter (12430): #1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:159)
E/flutter (12430): <asynchronous suspension>
E/flutter (12430): #2      TextureAndroidViewController._sendCreateMessage (package:flutter/src/services/platform_views.dart:1037)
E/flutter (12430): <asynchronous suspension>
E/flutter (12430): #3      AndroidViewController.create (package:flutter/src/services/platform_views.dart:743)
E/flutter (12430): <asynchronous suspension>
E/flutter (12430): #4      RenderAndroidView._sizePlatformView (package:flutter/src/rendering/platform_view.dart:195)
E/flutter (12430): <asynchronous suspension>
E/flutter (12430):

```

**Flutter information**
```
[√] Flutter (Channel beta, 1.25.0-8.3.pre, on Microsoft Windows [Version 10.0.19041.746], locale en-US)
    • Flutter version 1.25.0-8.3.pre at E:\flutter
    • Framework revision 5d36f2e7f5 (3 weeks ago), 2021-01-14 15:57:49 -0800
    • Engine revision 7a8f8ca02c
    • Dart version 2.12.0 (build 2.12.0-133.7.beta)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at C:\Users\Sreelal\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: E:\Apps\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Android Studio (version 4.1.0)
    • Android Studio at E:\Apps\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (2 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 88.0.4324.104
    • Edge (web)   • edge   • web-javascript • Microsoft Edge 85.0.564.67

• No issues found!
```

**Device (please complete the following information):**
 - Device: OnePlus Nord
 - OS: Oxygen OS 10.5.10.AC01DA (Android 10)

**Additional context**
The issue only exists in the release mode.

**Comment by tiloc** [comment]
Just a thought: your release build looks obfuscated by R8. Maybe you need to exclude the net.touchcap.** classes with a proguard-rules file?

**Comment by juliansteenbakker** [comment]
Fixed in #255
